pkg/operator/operator.go

```diff
@@ -226,46 +206,23 @@ func (c *Controller) syncHandler(key string) error {
 		c.logger.Log("error", err.Error())
 		return err
 	}
-
-	helmClient, err := c.getHelmClientForRelease(*hr)
+	err = c.release.Sync(hr.DeepCopy())
 	if err != nil {
-		c.logger.Log("warning", err.Error(), "resource", hr.ResourceID().String())
-		return nil
+		c.recorder.Event(hr, corev1.EventTypeWarning, ReleaseSynced,
```

Comment:
Why not 2 different event reasons?

Reply:
That is an option too. Naming suggestions?

Reply:
`FailedReleaseSync`, in the same fashion as `FailedScheduling`, `FailedMount`, etc.